lib/src/suggestor_mixins.dart

```diff
@@ -42,37 +44,25 @@ import 'suggestor.dart';
 ///         }
 ///       }
 ///     }
-mixin AstVisitingSuggestor<R> on AstVisitor<R> implements Suggestor {
-  final _patches = <Patch>{};
+mixin AstVisitingSuggestor<R> on AstVisitor<R> {
+  final patches = <Patch>{};
 
-  /// The context helper for the file for which patches are currently being
-  /// generated.
-  FileContext get context => _context;
-  FileContext _context;
-
-  @override
-  Stream<Patch> generatePatches(FileContext context) async* {
-    _patches.clear();
-    _context = context;
+  bool shouldResolveAst(FileContext context) => false;
```

Comment:
Looks like the doc comment got removed; was this intentional?

Reply:
Whoops, nope not intentional. Will add back